packages/ckb-sdk-rpc/types/CKB_RPC/index.d.ts

```diff
@@ -117,6 +117,7 @@ declare module CKB_RPC {
   export interface NodeInfo {
     addresses: { address: string; score: string }[]
     node_id: string
+    is_outbound: any
```

Comment:
Isn't a `boolean | null` ?

Reply:
updated in https://github.com/nervosnetwork/ckb-sdk-js/pull/180/commits/9d917b4a7de87904f823890cbc253379a05701b1